### PR TITLE
[WIP] Sync/jetpack related posts

### PR DESCRIPTION
**Pull request by retrofox**
[WIP]

### CODE REVIEW

File: modules/related-posts/jetpack-related-posts.php [hunk 1]
```diff
@@ -72,7 +73,11 @@ public function __construct( $blog_id_local, $blog_id_wpcom ) {
 		add_action( 'wp', array( $this, 'action_frontend_init' ) );
 
 		if ( ! class_exists( 'Jetpack_Media_Summary' ) ) {
-			jetpack_require_lib( 'class.media-summary' );
+			$lib = 'class.media-summary';
```
Comment (retrofox) [inline modules/related-posts/jetpack-related-posts.php]: Requiring library depending on the platform.

File: modules/related-posts/jetpack-related-posts.php [hunk 2]
```diff
@@ -81,6 +86,24 @@ public function __construct( $blog_id_local, $blog_id_wpcom ) {
 		}
 	}
 
+	protected function set_citations_enabled() {
+		//disable, some bugs causing query parsing errors
+		return;
```
Comment (retrofox) [inline modules/related-posts/jetpack-related-posts.php]: it's like we should remove this function :-/

File: modules/related-posts/jetpack-related-posts.php [hunk 3]
```diff
@@ -116,6 +139,7 @@ public function action_admin_init() {
 	 * @returns null
 	 */
 	public function action_frontend_init() {
+		$this->set_citations_enabled();
```
Comment (retrofox) [inline modules/related-posts/jetpack-related-posts.php]: `set_citations_enabled()` is a `noop()` method.

**Comment by brbrr** [comment]
@retrofox this PR needs rebase, and since it's staled - is it still needed?

**Comment by retrofox** [comment]
> is it still needed?

I don't think so. Let's close it and opening later if it's needed.
Thanks Yaroslav.

**Pull request closed without merging**